static/app/views/performance/http/httpSamplesPanel.tsx

```diff
@@ -80,6 +82,28 @@ export function HTTPSamplesPanel() {
     referrer: 'api.starfish.http-module-samples-panel-metrics-ribbon',
   });
 
+  const {
+    data: responseCodeData,
+    isFetching: isResponseCodeDataLoading,
+    error: responseCodeDataError,
+  } = useSpanMetrics({
+    search: MutableSearch.fromQueryObject(filters),
+    fields: ['span.status_code', 'count()'],
+    sorts: [{field: 'span.status_code', kind: 'asc'}],
+    enabled: isPanelOpen,
+    referrer: 'api.starfish.http-module-samples-panel-response-bar-chart',
+  });
+
+  const responseCodeBarChartSeries: Series = {
+    seriesName: 'span.status_code',
+    data: (responseCodeData ?? []).map(item => {
+      return {
+        name: item['span.status_code'] || t('N/A'),
```

Comment:
should we be more opinionated about what response codes we care about? This could potentially make the chart noisy if there are 10+ status codes.

Perhaps we can add filters to the chart that all you to show `2xx`, `3xx`, `5xx`?

Reply:
Good question, I was thinking the same thing! I'm torn on this, I think this might be a good place to show everything and let people decide what's important. I feel like even 10+ is still okay, and useful to know, but I want to see what people say when they get eyes on this